src/services/cronJobs/syncUserPoewrs.ts

```diff
@@ -2,14 +2,16 @@ import Bull from 'bull';
 import config from '../../config';
```

Comment:
Just noticed that this TS file is named wrong.

Currently: syncUserPoewrs.ts
Should be: syncUserPowers.ts

Reply:
Oh right